javascripts/discourse/initializers/discourse-image-annotator.js.es6

```diff
@@ -1,24 +1,56 @@
     const appEvents = container.lookup("service:app-events");
 
     //Display markerjs when image is clicked
-    function showMarkerJs(img, config) {
-      let mark = new markerjs2.MarkerArea(img, { previousState: config });
-      let editor = document.querySelector("#reply-control textarea.d-editor-input").value;
-      let images = document.querySelectorAll(".d-editor-preview img");
+    function showMarkerJs(img) {
+      let mark = new markerjs2.MarkerArea(img);
+
+      //Exec after marker's toolbar checkmark is clicked
+      mark.settings.displayMode = "popup";
+      mark.addRenderEventListener((dataUrl) => {
+        if (mark.getState().markers.length > 0) {
+          getNewImgUrl(img, dataUrl);
```

Comment:
Here you are using the old function name. Also make sure to camel case `SaveNewImg` to `saveNewImg`